frontend/src/breeding_list.tsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * @file This file contains the BreedingList function. This function fetches
+ *       breeding information for a given herd, and displays a list and edit
+ *       form to handle breedings.
+ */
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles';
+
+import { get } from '@app/communication';
+import { useMessageContext } from '@app/message_context';
+import { useDataContext } from './data_context';
+import { SortedTable, Column } from './sorted_table';
+import { Typography } from '@material-ui/core';
+import { BreedingForm } from './breeding_form';
+
+const useStyles = makeStyles({
+  breeding: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+  },
+  table: {
+    padding: '10px 0',
+    margin: 0,
+    transition: 'width .5s',
+  },
+  form: {
+    padding: '10px 0',
+    margin: 0,
+    border: '1px solid lightgrey',
+    borderRadius: '3px',
+    overflowX: 'hidden',
+    transition: 'width .5s',
+  },
+  loading: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+  header: {
+    fontSize: "1.2em",
+  }
+});
+
+export interface Breeding {
+  id?: number,
+  breed_date: string,
+  breed_notes: string,
+  father: string,
+  mother: string,
+  birth_date: string,
+  birth_notes: string,
+  litter_size: number,
+}
+
+/**
+ * The BreedingList function. This function fetches breeding information for a
+ * given herd, and displays a list and edit form to handle breedings.
+ */
+export function BreedingList({id}: {id: string | undefined}) {
+  const [ breedingEvents, setBreedingEvents ] = React.useState([] as Breeding[])
+  const [ active, setActive ] = React.useState(null as any)
+  const { userMessage } = useMessageContext()
+  const { genebanks } = useDataContext()
+  const style = useStyles();
+
+  // Parent information from the genebank
+  const parents = React.useMemo(() => {
+    const parentNumbers = breedingEvents.flatMap(b => [b.mother, b.father]);
+    return genebanks.flatMap(genebank => {
+      return genebank.individuals.filter(i => parentNumbers.includes(i.number))
+    })
+  }, [genebanks, breedingEvents])
+
+  const columns: Column[] = [
+    {field: 'breed_date', label: 'Parningsdatum', sortAs: 'date', hidden: false},
+    {field: 'breed_notes', label: 'Parningsanteckningar', hidden: true},
+    {field: 'mother', label: 'Moder', sortAs: 'numbers', hidden: false},
+    {field: 'father', label: 'Fader', sortAs: 'numbers', hidden: false},
+    {field: 'birth_date', sortAs: 'date', label: 'Födslodatum', hidden: false},
```

Comment:
```suggestion
    {field: 'birth_date', sortAs: 'date', label: 'Födelsedatum', hidden: false},
```

This is used elsewhere in the page